internal/slicer/slicer_test.go

```diff
@@ -320,33 +320,33 @@ var slicerTests = []slicerTest{{
 		"/file":     "file 0644 fc02ca0e {other-package_myslice}",
 	},
 }, {
-	summary: "Install two packages, explicit path has preference over implicit parent",
+	summary: "Install two packages, explicit path has preference over implicit parent (alphabetical order)",
```

Comment:
Would merging the two tests with the following packages work?

```
a-implicit-parent-1  -> lists /dir/file-1 only
b-explicit-dir       -> lists /dir only
c-implicit-parent-2  -> lists /dir/file-2 only
```

Reply:
Great idea! I have changed the test to reflect that, thanks!